lib/puppet/util/windows/string.rb

```diff
@@ -9,6 +9,10 @@ def wide_string(str)
     # causing buffer overruns.
     #
     # See http://svn.ruby-lang.org/cgi-bin/viewvc.cgi?revision=41920&view=revision
+    unless str.is_a? String
```

Comment:
I figured out why you were dealing with the symbols - for some reason we use symbols as hash keys in the specs and other places (even though it doesn't make a lot of sense TBH).

That can be addressed elsewhere - let's leave this method alone for now.